bundles/org.openhab.binding.touchwand/src/main/java/org/openhab/binding/touchwand/internal/dto/TouchWandUnitDataWallController.java

```diff
@@ -25,8 +25,15 @@
 
     private CurrStatus currStatus = new CurrStatus();
 
+    // currStatus actually can be null since the object is created by gson fromJson
+    // in case the key is null or not exist , the variable will be null.
+
+    @SuppressWarnings({ "unused", "null" })
```

Comment:
To avoid this, DTOs shouldn't be annotated with NonNullByDefault, because it gives a false sense of safety.

Reply:
done - annotation removed